draft-ietf-tls-certificate-compression.md

```diff
@@ -146,6 +146,10 @@ After decompression, the Certificate message MUST be processed as if it were
 encoded without being compressed.  This way, the parsing and the verification
 have the same security properties as they would have in TLS normally.
 
+In order for certificate compression to function correctly, the underlying
+compression algorithm MUST be determenistic and it MUST output the same data
```

Comment:
deterministic

perhaps "no compressed data can map to either two inputs or two outpots"

Reply:
Fixed the typo.

I'm not sure what "compressed data map to inputs" mean?  I think the requirements on compression functions f/f^{-1} that we want is f^{-1}(f(x)) = x for all x, which I feel like I've stated here.

Reply:
You could add that there shouldn't be compression collisions (which is a term that was floated during the session last week).